src/css/homepage.css

```diff
@@ -89,3 +140,56 @@
 .homepage-shape-links-container > img {
     width: 10%;
 }
+
+/* Homepage theme park shape CSS & animation */
+
+.homepage-shape-links-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 2em;
+    background-image: linear-gradient(white, white 30%, white 40%, dodgerBlue 100%);
+}
+
+.homepage-shape-links-container > img {
+    width: 10%;
+    padding-bottom: 2em;
+
+}
+
+.homepage-shape-links-container-img{
+    display: flex;
+}
+
+.homepage-shape-links-container-img:hover{
+    animation: shape-img-ani-1 2s ease 0s 1 normal forwards;
+  }
+  @keyframes shape-img-ani-1 {
```

Comment:
I encourage either to:
- use a preset animation as it will have both behaviors for "entering" the animation and "exiting" it
- add the process backwards for the animation, after hovering the images goes right back to the original state

Also, fix indentation for this, please.